manifests/g/Google/Terraformer/0.8.24/Google.Terraformer.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.Terraformer
+PackageVersion: 0.8.24
+Installers:
+- Architecture: x64
+  InstallerType: portable
+  InstallerUrl: https://github.com/GoogleCloudPlatform/terraformer/releases/download/0.8.24/terraformer-all-windows-amd64
+  InstallerSha256: F69C09F7045A6010B892BF2F32229F3B5AE3F20F54FF975F2CBDEDCEE5CB0241
+  Commands:
+  - terraformer

```

Comment:
This doesn't seem to work?

Reply:
It works for me. It probably has the same issue as #111454 where the base executable isn't renamed if symlinks aren't allowed

```
--> Installing the Manifest 0.8.24

Found Terraformer [Google.Terraformer] Version 0.8.24
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/GoogleCloudPlatform/terraformer/releases/download/0.8.24/terraformer-all-windows-amd64
  ██████████████████████████████   410 MB /  410 MB
Successfully verified installer hash
Starting package install...
Command line alias added: "terraformer"
Path environment variable modified; restart your shell to use the new value.
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName DisplayVersion Publisher ProductCode
----------- -------------- --------- -----------
Terraformer 0.8.24         Google    Google.Terraformer__DefaultSource



PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> terraformer
Usage:
   [command]

Available Commands:
  help        Help about any command
  import      Import current state to Terraform configuration
  plan        Plan to import current state to Terraform configuration
  version     Print the version number of Terraformer

Flags:
  -h, --help      help for this command
  -v, --version   version for this command

Use " [command] --help" for more information about a command.
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> terraformer -v
version v0.8.24
```
<!--
[Policy] Reset Feedback
-->